packages/playwright-ct-core/src/vitePlugin.ts

```diff
@@ -267,35 +257,13 @@ async function checkSources(buildInfo: BuildInfo): Promise<boolean> {
   return false;
 }
 
-async function checkNewTests(suite: Suite, buildInfo: BuildInfo, componentRegistry: ComponentRegistry): Promise<boolean> {
-  const testFiles = new Set<string>();
-  for (const project of suite.suites) {
-    for (const file of project.suites)
-      testFiles.add(file.location!.file);
+async function populateComponentsFromTests(componentRegistry: ComponentRegistry, componentsByTestFile: Map<string, string[]>) {
+  const importInfos: Map<string, ImportInfo[]> = await getUserData('playwright-ct-core');
+  for (const [testFile, importList] of importInfos) {
```

Comment:
IIUC, this is not necessary the `testFile`, could be any imported dependency of the test file? Let's be more precise with the naming to avoid confusion in the future.

Reply:
Same as above!